build/config/compiler/BUILD.gn

```diff
@@ -668,6 +618,20 @@ config("runtime_library") {
       libs += [ "atomic" ]
     }
   }
+
+  # Linux standard liburary setup.
+  if (is_linux) {
+    cflags_cc += [
+      "-stdlib=libc++"
+    ]
+    ldflags += [
+      "-stdlib=libc++",
+      "-static-libstdc++",
+      "-Wl,-Bstatic", "-lc++abi",
+      "-Wl,-Bdynamic",
```

Comment:
nit: please put this up on the previous line, as it's really resetting state from that.